src/components/NoticeText/CampaignRegistration10DlcNoticeText.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const CampaignRegistration10DlcNoticeText: React.FC = () => (
+  <>
+    <p>
+      <b>
+        You have started but not yet completed your 10DLC registration form.
+      </b>
+      <br /> <br />
+      You must complete registration in order to gain access to our{" "}
+      <a
+        href="https://www.politicsrewired.com/pricing"
+        target="_blank"
+        rel="noreferrer"
+      >
+        Standard pricing plan
+      </a>
+      . Please complete the second section of our 10DLC form to finalize your
+      registration.
+      <br /> <br />
+      You are currently on our{" "}
+      <a
+        href="https://www.politicsrewired.com/pricing"
+        target="_blank"
+        rel="noreferrer"
+      >
+        Quick Start pricing plan
+      </a>
+      . You may send messages without registering for 10DLC at an increased
+      price per message segment.
+    </p>
+    <p>
+      To learn more about 10DLC and our pricing plans, please see our{" "}
+      <a
+        href="https://docs.spokerewired.com/article/124-10dlc"
+        target="_blank"
+        rel="noreferrer"
+      >
+        10DLC knowledge base article
+      </a>
+      .
```

Comment:
question: would it make sense to extract this out into a common component?

Reply:
yeah i was thinking this too and just got lazy 👍🏾